src/lib.rs

```diff
@@ -8,18 +8,31 @@ mod visit_mut;
 use ident_visitor::IdentVisitor;
 use proc_macro2::TokenStream;
 use quote::quote;
+use serde_json::json;
+use std::collections::HashMap;
 use std::fs::File;
 use std::io::prelude::*;
 use syn::visit_mut::VisitMut;
 
+const PLACEHOLDER: &str = "PLACEHOLDER_";
+
 const OUTPUT_FILE: &str = "representation.rs";
```

Comment:
Looking at the docs, I think this is supposed to be called `representation.txt`. Not sure if the file extension is important for it to be picked up.

Reply:
Yes but the run.sh that is ran by the docker make a fmt on the .rs before converting it to .txt. I think to have a fully Rust solution we could run the fmt in the rust code directly but it might be too much for this PR

Reply:
Got it, thanks! It's fine the way it is.